src/main/java/org/springframework/data/redis/connection/RedisZSetCommands.java

```diff
@@ -740,4 +745,34 @@ default Long zRemRangeByScore(byte[] key, double min, double max) {
 	 */
 	Set<byte[]> zRangeByLex(byte[] key, Range range, Limit limit);
 
+	/**
+	 * Convert tuples to map of bytes and double.
+	 * Bytes represents the value of the element and double is for the score.
+	 * @param tuples
+	 * @return
+	 */
+	default Map<byte[], Double> zAddArgs(Set<Tuple> tuples) {
```

Comment:
This interface is driver-agnostic and the change would affect users using the Lettuce driver. Rather move the tuple conversion into `JedisConverters`.

Reply:
Thanks for the comment. Yes, it's better to be in JedisConverters. I've made the changes. Please check